routes/index.js

```diff
@@ -1,7 +1,26 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
+const { Movie } = require("../models/Movie.model");
 
 /* GET home page */
-router.get('/', (req, res, next) => res.render('index'));
+router.get("/", (req, res, next) => res.render("index"));
+
+router.get("/movies", async (req, res, next) => {
+  const movies = await Movie.find();
+  res.render("movies", { movies });
+});
+
+router.get("/movie/:id", async (req, res, next) => {
+console.log(req.params.id);
```

Comment:
this is not properly indented, you might want to setup prettier in vscode for autoformat on save :)